Testing/SystemTests/tests/analysis/ISISIndirectInelastic.py

```diff
@@ -227,6 +227,19 @@ def get_reference_files(self):
         return ["II.TOSCAReductionFromFile.nxs"]
 
 
+class TOSCASimpleReductionUsingOldRunNumber(ISISIndirectInelasticReduction):
```

Comment:
I can't seem to work out what this does, but I believe these don't run on linux variants or macOS, since RHEL7 is our main tester for these I don't think these would run for a normal dev to see during a PR. Would it be possible to make a unit test specifically for the TOSCA data that this PR is attempting to fix, somewhere else? If not then here is about as good as we can get.

Reply:
I think this is the best place for this test